dotcom-rendering/src/web/components/GetMatchNav.tsx

```diff
@@ -18,6 +18,8 @@ export const GetMatchNav = ({ matchUrl }: Props) => {
 		minByMinUrl?: string;
 	}>(matchUrl);
 
+	const MATCH_URL: string = matchUrl;
```

Comment:
Is there a reason for defining this as it's own variable?

Reply:
Yeah, MatchNav cannot get direct access so it needs to be passed as a variable.